packages/Tooltip/index.tsx

```diff
@@ -133,7 +133,7 @@ export type PlacementOptions =
   | 'left-start'
 
 export interface TooltipOptions {
-  children: React.ReactElement
+  children: React.ReactElement | React.ReactNode
```

Comment:
I'm pretty sure that `ReactElement` is in `ReactNode`
```suggestion
  children: React.ReactNode
```

Reply:
You're absolutely right sire, thanks @Calvein